src/WireMock.Net/Http/ByteArrayContentHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Net.Http;
+using System.Net.Http.Headers;
+using JetBrains.Annotations;
+using WireMock.Validation;
+
+namespace WireMock.Http
+{
+    internal static class ByteArrayContentHelper
+    {
+        /// <summary>
+        /// Creates a ByteArrayContent object.
+        /// </summary>
+        /// <param name="content">The byte[] content (cannot be null)</param>
+        /// <param name="contentType">The ContentType (can be null)</param>
+        /// <returns>ByteArrayContent</returns>
+        internal static ByteArrayContent Create([NotNull] byte[] content, [CanBeNull] MediaTypeHeaderValue contentType)
```

Comment:
I use nullable reference types feature in the modern C#, where it is available, it rocks. Just a note.

Reply:
Yes. I started using the new features for some small new projects on GitHub.
However, I did not yet check this project. Will take some time to fix all warnings I think.

Reply:
Yes, it will. And sometimes it reveals some problems or requires a chain of modifications, but, ultimately, you have a good feeling that the codebase became more reliable and organized, I like it :)